i18n/en.json

```diff
@@ -0,0 +1,3 @@
+{
+	"placenewsection-desc": "Adds the <nowiki>__ADDNEWSECTIONBELOW__</nowiki> and <nowiki>__ADDNEWSECTIONABOVE__</nowiki> magic words to control new section placement\""
+}
```

Comment:
@reedy Are missing the missing LFs at EOF in these files any cause for concern, from your perspective?

I assume it's benign in this context, but I can think of some situations (in a Unix environment) where it's not.